api/api/serializers/team_set.py

```diff
@@ -9,8 +9,11 @@ class TeamSetSerializer:
     """
 
     def __init__(self, team_set: TeamSet):
-        pass
+        self.team_set = team_set
 
     @property
     def data(self) -> Dict[str, Any]:
-        pass
+        return {
+            "num_teams": self.team_set.num_teams,
+            "teams": [team.todict() for team in self.team_set.teams],
```

Comment:
instead of team.todict(), there would just be a TeamSerializer, this is what Seth and Justin are working on

Reply:
I'll put down a TODO to update later